pipeline_steps/multi_node_aio_prepare.groovy

```diff
@@ -2,6 +2,8 @@ def prepare() {
   common.conditionalStage(
     stage_name: 'Prepare Multi-Node AIO',
     stage: {
+      common.prepareRpcGit(env.RPC_BRANCH)
```

Comment:
I left it there to try to preserve a degree of separation between the components but, given interdependent everything is, it probably provides no benefit. I'll remove it.